src/modules/MouseUtils/FindMyMouse/dllmain.cpp

```diff
@@ -192,7 +192,7 @@ void FindMyMouse::parse_settings(PowerToysSettings::PowerToyValues& settings)
             }
             else
             {
-                throw;
+                throw std::exception();
```

Comment:
For all such cases: let's throw `std::runtime_error("<error description>")` to make it a bit more informative.